flink-dstl/flink-dstl-dfs/src/main/java/org/apache/flink/changelog/fs/FsStateChangelogWriter.java

```diff
@@ -369,7 +390,6 @@ public void confirm(SequenceNumber from, SequenceNumber to, long checkpointId) {
                             changelogRegistry.stopTracking(localHandle);
                             localChangelogRegistry.register(localHandle, checkpointId);
```

Comment:
since already register local handle in persist method , this line can be deleted.

Reply:
Fixed.